src/campsite-info/portage-info.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from 'react';
+import { CampHeader } from './components/camp-header';
+import { CampImages } from './components/camp-images';
+import { Box } from '@mui/material';
```

Comment:
let's try to keep all the npm package imports above the local file system imports